ecosystem/sep-0008.md

```diff
@@ -117,7 +117,7 @@ message|string|A human readable string containing information to pass on to the
 
 #### Action Required Response
 
-An Action Required response will have a `200` HTTP status code and `action_required` as the `status` value. It means that the user must complete an action before this transaction can be approved. The approval service will provide a URL that facilitates the action. Upon completion, the user will resubmit the transaction.
+An Action Required response will have a `200` HTTP status code and `action_required` as the `status` value. It means that the user must complete an action before this transaction can be approved. The approval service will provide a URL that facilitates the action. Upon completion, the user will resubmit the transaction if it's synchronous (the default) or wait for a confirmation from the approval service to continue.
```

Comment:
What does the waiting process look like? Does a client wait for some signal? What is that signal?

Reply:
In our case, the user get's an email that she has passed kyc and then she can trade. The client is not involved here. It basically means that afterwards the client does nothing.

Reply:
Got it, so the app if it is synchronous would return to resending the same request again and retrying after the action required step, where-as if it is not synchronous the app would stop. Is that an accurate understanding?

Reply:
That's up to the implementation, I'm not sure how current clients handle that, but if it's synchronous, then yes, the client could do that. 

Maybe we can leave that out for now as well, I'm not aware of other SEP-8 implementations so I'm not sure if that is actually a thing.